ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Ledger/Query.hs

```diff
@@ -401,8 +404,8 @@ instance (ShelleyCompatible proto era, ProtoCrypto proto ~ crypto) => QueryLedge
           SL.queryDRepState st drepCreds
         GetDRepStakeDistr dreps ->
           SL.queryDRepStakeDistr st dreps
-        GetCommitteeState ->
```

Comment:
`queryCommitteeState` is now deprecated in cardano-ledger. 

I did read the comment about not removing queries, however, I thought that since this is not in mainnet, it's maybe acceptable to replace it in this way (also I'll be integrating this change in cardano node). 

I'm happy to change it to do it properly, if you think it's necessary.

Reply:
As the query is guarded by an experimental version, it is fine to just remove it :+1: